SharedCode/src/commonMain/kotlin/ApplicationContract.kt

```diff
@@ -4,10 +4,16 @@ import kotlinx.coroutines.CoroutineScope
 
 interface ApplicationContract {
     interface View {
-        fun setLabel(text: String)
+        fun populateOriginAndDestinationSpinners(stations: List<String>)
+        fun setStationSubmitButtonHandler()
+        fun setStationSubmitButtonText(text: String)
+        fun getOriginStation() : String
+        fun getDestinationStation() : String
```

Comment:
I would expect no space before a colon here (but I would expect a single space after as you have here)